tests/integration/pagecontainer/page_load_failure_core.js

```diff
@@ -15,13 +15,13 @@ $( document ).on( "pagecontainerloadfailed", function( event ) {
 				$( "#go-to-other-page" ).click();
 			},
 			function() {
-				deepEqual( $.mobile.pageContainer.pagecontainer( "getActivePage" )
+				deepEqual( $( ".ui-pagecontainer" ).pagecontainer( "getActivePage" )
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?